src/backend/distributed/commands/schema.c

```diff
@@ -186,3 +241,41 @@ FilterDistributedSchemas(List *schemas)
 
 	return distributedSchemas;
 }
+
+
+/*
+ * EnsureSequentialModeForSchemaDDL makes sure that the current transaction is already in
+ * sequential mode, or can still safely be put in sequential mode, it errors if that is
+ * not possible. The error contains information for the user to retry the transaction with
+ * sequential mode set from the begining.
+ *
+ * Copy-pasted from type.c
+ */
+static void
+EnsureSequentialModeForSchemaDDL(void)
+{
+	if (!IsTransactionBlock())
+	{
+		/* we do not need to switch to sequential mode if we are not in a transaction */
+		return;
+	}
+
+	if (ParallelQueryExecutedInTransaction())
+	{
+		ereport(ERROR, (errmsg("cannot create or modify type because there was a "
```

Comment:
It should be `schema` instead of `type`.

Reply:
copy-paste's gift :)